test/srcs/testNick.cpp

```diff
@@ -45,6 +57,23 @@ void valid_nick_should_void(Server& s)
     }
 }
 
+void valid_nick2_should_void(Server& s)
+{
+    try {
+
+        TEST_SETUP(test, s, 1);
+        TcpSocket& so = *sockets.at(0);
+        send_line(so, validPassMsg);
+        send_line(so, validUserMsg);
+
+        // test
+        send_line(so, "NICK ro3ro\r\n");
```

Comment:
il manque une assertion type
```suggestion
        send_line(so, "NICK ro3ro\r\n");
        std::string reply = recv_lines(so);
        AssertReply ar(reply);
        ar.is_empty();
```